cSpell.json

```diff
@@ -1058,6 +1058,13 @@
                  "containerd"
             ]
         }
+    ,
+        {
+            "filename": "**/specification/app/resource-manager/Microsoft.App/**/*.json",
+            "words": [
+                "Loglevel"
```

Comment:
>Loglevel [](http://example.com/codeflow?start=17&length=8)

Do you know why it is treating as spell error? Seems we are using logLevel/LogLevel so should be correct.

Reply:
oh, sorry. I use `Loglevel` term before on the spec file, it should be `LogLevel`.  So no need to add new words on cspell file.